### PR TITLE
fix: valentine issue

### LINKED ISSUE #1: Broken heart 💔
# Why?
Improperly closed div with class 'heart' causes incorrect shadow appearance.
```html
<div class="heart" </div>
```

# Possible solution
Close tag and move `.hearts` inside `.card`

### PR DESCRIPTION
**Pull request by HobMartin**
# Why
This PR will resolve #1 issue and improve CSS by moving hover animation from JS to CSS. It will work on mobile devices as well

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
-        <div class="hearts">
-            <div class="one"></div>
-            <div class="two"></div>
-            <div class="three"></div>
-            <div class="four"></div>
-            <div class="five"></div>
-        </div>
-    </div>
-    <div class="shadow"></div>
-    <script src="script.js"></script>
```
Comment (andruha1501) [inline index.html:29]: why did you remove script?

**Review by andruha1501 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (HobMartin) [reply index.html:29]: Logic implemented by CSS

**Review by andruha1501 — APPROVED** [review]
